### PR TITLE
Ingestion: inlg-chal 2022

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Comment by evanmiltenburg** [comment]
> Build successful. Some useful links:
> 
> * Complete site preview: https://preview.aclanthology.org/inlg-chal-22-ingestion
> * Potential volumes of interest: [2022.inlg-main](https://preview.aclanthology.org/inlg-chal-22-ingestion/volumes/2022.inlg-main), [2022.inlg-demos](https://preview.aclanthology.org/inlg-chal-22-ingestion/volumes/2022.inlg-demos), [2022.inlg-genchal](https://preview.aclanthology.org/inlg-chal-22-ingestion/volumes/2022.inlg-genchal), [2022.inlg-inlg-2022](https://preview.aclanthology.org/inlg-chal-22-ingestion/volumes/2022.inlg-inlg-2022)
> 
> This preview will be removed when the branch is merged.

Thanks for this! I'm following along to better understand the ingestion process. It seems that this link does not work:[2022.inlg-inlg-2022](https://preview.aclanthology.org/inlg-chal-22-ingestion/volumes/2022.inlg-inlg-2022). The others do seem to work.

File: data/yaml/name_variants.yaml [hunk 1]
```diff
@@ -5511,6 +5511,9 @@
 - canonical: {first: Yang, last: Liu}
   comment: 3M Health Information Systems
   id: yang-liu-3m
+- canonical: {first: Yang, last: Liu}
+  comment: other
+  id: yang-liu-other
```
Comment (mjpost) [inline data/yaml/name_variants.yaml:5516]: For "unknown"s, I think we should just use [the base name](https://aclanthology.org/people/yang-liu), i.e., `id: yang-liu`.

**Comment by mjpost** [comment]
> > Build successful. Some useful links:
> > 
> > * Complete site preview: https://preview.aclanthology.org/inlg-chal-22-ingestion
> > * Potential volumes of interest: [2022.inlg-main](https://preview.aclanthology.org/inlg-chal-22-ingestion/volumes/2022.inlg-main), [2022.inlg-demos](https://preview.aclanthology.org/inlg-chal-22-ingestion/volumes/2022.inlg-demos), [2022.inlg-genchal](https://preview.aclanthology.org/inlg-chal-22-ingestion/volumes/2022.inlg-genchal), [2022.inlg-inlg-2022](https://preview.aclanthology.org/inlg-chal-22-ingestion/volumes/2022.inlg-inlg-2022)
> > 
> > This preview will be removed when the branch is merged.
> 
> Thanks for this! I'm following along to better understand the ingestion process. It seems that this link does not work:[2022.inlg-inlg-2022](https://preview.aclanthology.org/inlg-chal-22-ingestion/volumes/2022.inlg-inlg-2022). The others do seem to work.

@evanmiltenburg that link is a bug that I just fixed in another branch (https://github.com/acl-org/acl-anthology/pull/2323/commits/5a4cb43144bbe1225ee19409ff8ea8d414702fd8). It occurred because we used to just have `<volume>` nodes underneath `<collection>`, but recently added an `<event>` child.

**Comment by Tirthankar-Ghosal** [comment]
Hi! Any update on when the INLG 2022 GenChal proceedings would be live?

**Comment by evanmiltenburg** [comment]
My impression is that the ingestion is stuck in the reviewing phase, but I guess the timing wasn't ideal (submitting before the Christmas holidays).

**Review by mjpost — APPROVED** [review]

**Pull request merged**